website/templates/project/modal_mailing_list_contributors.mako

```diff
@@ -0,0 +1,64 @@
+<div class="modal fade" id="mailingListContributorsModal">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+              <h3>Project Mailing List Email:</h3>
+            </div>
+
+            <div class="modal-body">
+                <h4 class="row text-center">
+                <div class="btn-group">
+                    <button data-clipboard-text="${node['id']}@osf.io" title="Copy to clipboard" class="btn btn-default btn-sm m-r-xs copy-button zeroclipboard-is-hover" style="margin-right: 0px">
+                        <i class="fa fa-copy"></i>
+                    </button>
```

Comment:
Most modern browsers don't support it, but I was asked to include it anyway. It's the same as other "Hey users, this is something you should copy"-textboxes around the site.